OneWelcomeExampleApp/ViewControllers/Login/LoginViewController.swift

```diff
@@ -30,9 +30,7 @@ class LoginViewController: UIViewController {
 
     var profiles = [UserProfile]() {
         didSet {
-            if let tableView = profilesTableView {
-                tableView.reloadData()
-            }
+            reloadProfiles()
```

Comment:
why not just `profilesTableView?.reloadData()`?

Reply:
well I just moved the code to the more proper place.
moreover it's unwrapped that way.